src/simtools/reporting/docs_auto_report_generator.py

```diff
@@ -215,3 +219,107 @@ def auto_generate_observatory_reports(self):
         """Generate all observatory reports based on which --all_* flags are passed."""
         for params in self._generate_observatory_report_combinations():
             self._generate_single_observatory_report(*params)
+
+    def auto_generate_simulation_configuration_reports(self):
+        """Generate simulation configuration reports for one or all model versions.
+
+        If --all_model_versions is set, produce reports for every model version in
+        the DB; otherwise produce reports only for the configured model_version.
+        """
+        model_versions = (
+            self.db.get_model_versions()
+            if self.args.get("all_model_versions")
+            else [self.args["model_version"]]
+        )
+
+        for version in model_versions:
+            # update args and create a per-version output directory
+            self.args.update({"model_version": version})
+            output_path = Path(self.output_path) / str(version)
+
+            ReadParameters(
+                self.db_config, self.args, output_path
+            ).produce_simulation_configuration_report()
+
+            logger.info(f"Configuration reports for (v{version}) produced: {output_path}")
+
+    def auto_generate_calibration_reports(self):
+        """Generate calibration reports for one or all model versions.
+
+        Mirrors the pattern used by other auto_generate_* methods: if
+        --all_model_versions is set, produce reports for every model version in
+        the DB; otherwise produce reports only for the configured model_version.
+        """
+        model_versions = (
```

Comment:
Lines 253 to 262 are exactly identical to lines 229 to 238. Look at it - and decide yourself if you want to change it.